wagtail/admin/navigation.py

```diff
@@ -1,45 +0,0 @@
-from django.conf import settings
-
-from wagtail.models import Page
-
-
-def get_pages_with_direct_explore_permission(user):
-    # Get all pages that the user has direct add/edit/publish/lock permission on
-    if user.is_superuser:
-        # superuser has implicit permission on the root node
-        return Page.objects.filter(depth=1)
-    else:
-        return Page.objects.filter(
-            group_permissions__group__in=user.groups.all(),
-            group_permissions__permission_type__in=["add", "edit", "publish", "lock"],
-        )
-
-
-def get_explorable_root_page(user):
```

Comment:
We need an upgrade path for these in the very least.
wagtail-localize, for example, uses `get_explorable_root_page` and I suspect other packages do too

Reply:
Addressed in https://github.com/wagtail/wagtail/pull/9594/commits/d20bf2f47443abfa787f7fce5e835dc16d4d0b31.